core-bundle/assets/controllers/choices-controller.js

```diff
@@ -1,21 +1,27 @@
 import { Controller } from '@hotwired/stimulus';
+import Choices from 'choices.js';
+import css from '!!css-loader?{"url": false}!../styles/component/choices.pcss';
```

Comment:
Side note: If we would fix the paths in our legacy css files to being file-relative, we could omit the `?{…}` notation here as well as the respective config in our `webpack.config.js`.

Reply:
:thinking: In fact we can omit this here anyways, because I'm not importing the default backend styles, anymore. Adjusted in bd8097567b0aa6f28876687e28d4fc7c6f7b6f35.

Reply:
> Side note: If we would fix the paths in our legacy css files to being file-relative, we could omit the ?{…} notation here as well as the respective config in our webpack.config.js.

IIRC right now, it is still needed due to the `system/themes/flexible` path, should we leave a note there?

Reply:
At least here, the config is not needed. So let's do this in a separate issue/PR.